website/src/app/pricing/page.tsx

```diff
@@ -0,0 +1,726 @@
+            change your plan or terminate your account.
+          </p>
+
+          <a id="when-billed" className="pt-8"></a>
+          <blockquote className="font-semibold text-md p-2 my-2 border-s-4 border-neutral-300">
+            <p>When will I be billed?</p>
+          </blockquote>
+          <p className="mb-8">
+            When you start service, or at the beginning of each billing cycle.
+            Enterprise plans are billed quarterly or annually.
+          </p>
+
+          <a id="payment-methods" className="pt-8"></a>
+          <blockquote className="font-semibold text-md p-2 my-2 border-s-4 border-neutral-300">
+            <p>What payment methods are available?</p>
+          </blockquote>
+          <p className="mb-8">
+            The Starter plan is free and does not require a credit card to be
+            entered. Enterprise plans can be paid via credit card, ACH, or wire
+            transfer and will have a 100% discount applied for the duration of
+            the beta.
+          </p>
+
+          <a id="special-pricing" className="pt-8"></a>
+          <blockquote className="font-semibold text-md p-2 my-2 border-s-4 border-neutral-300">
+            <p>
+              Do you offer special pricing for nonprofits and educational
+              institutions?
+            </p>
+          </blockquote>
+          <p className="mb-8">
+            Yes. Not-for-profit organizations and educational institutions are
+            eligible for a 50% discount. Contact us (support@firezone.dev) to
+            apply for the discount.
+          </p>
+
+          <a id="enhance-cybersecurity" className="pt-8"></a>
+          <blockquote className="font-semibold text-md p-2 my-2 border-s-4 border-neutral-300">
+            <p>
+              Other than using Firezone, is there anything I can do to improve
+              my cybersecurity?
+            </p>
+          </blockquote>
+          <p className="mb-8">
+            Firezone helps protect your network and private resources, however,
+            organizations should consider a balanced security and risk
+            management posture that takes into account all the different parts
+            of their business.
+          </p>
+        </div>
+      </section>
+
+      <section className="bg-neutral-100 border-t border-neutral-200 p-14">
+        <div className="mx-auto max-w-screen-xl md:grid md:grid-cols-2">
+          <div>
+            <h2 className="w-full justify-center mb-8 text-2xl md:text-3xl font-semibold text-neutral-900">
+              The WireGuard® solution for Enterprise.
+            </h2>
+          </div>
+          <div className="mb-14 w-full text-center">
+            <Link href="/contact/sales">
+              <button
+                type="button"
+                className="w-64 text-white font-bold tracking-tight rounded duration-0 hover:scale-105 transition transform shadow-lg text-lg px-5 py-2.5 bg-primary-450"
```

Comment:
If this is intended just ignore this comment, but wanted to point out that this button is orange rather than the purple gradient of the other buttons on the page.

Reply:
Good catch -- yeah we have some of the more prominent CTA buttons orange just to stand out a little more